e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -157,11 +159,16 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
     dashboardDeals.filters.mainContainer.selectedFilters.statusReadyForChecker().contains(expectedText);
   });
 
-  it('renders only deals that have matching fields - MIA, AIN, Draft status, Ready for check status', () => {
+  it.only('renders only deals that have matching fields - MIA, AIN, Draft status, Ready for check status', () => {
```

Comment:
can this it.only be removed please

```suggestion
  it('renders only deals that have matching fields - MIA, AIN, Draft status, Ready for check status', () => {
```